### ISSUE TITLE
SA1607 inheritdoc analysis does not consider overrides

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1946]
The following code should report SA1607, but no warning is reported.

```csharp
/// <summary>
/// </summary>
/// <inheritdoc/>
public partial class ClassName
{
}
```

In addition, the following code should report SA1607, but no warning is reported.

```csharp
/// <content>
/// </content>
/// <inheritdoc/>
public partial class ClassName
{
}
```
